packages/devtools_app/lib/src/extensions/static/README.md

```diff
@@ -0,0 +1,5 @@
+## What are static extensions?
```

Comment:
What is the point of static extension? Why it is not just part of application?

Reply:
Good idea. Done.